x-pack/plugin/enrich/src/main/java/org/elasticsearch/xpack/enrich/AbstractEnrichProcessor.java

```diff
@@ -35,15 +35,31 @@
     protected final String matchField;
     protected final int maxMatches;
 
-    protected AbstractEnrichProcessor(String tag, Client client, String policyName, String field, String targetField,
```

Comment:
I prefer not putting all parameters on a new line and only adding a new line when it reaches the line length limit.

Reply:
For background on why it's configured this way: mostly I was guided by what Prettier (https://prettier.io/) does, whose strategy is broadly to try to fit everything on one like, but when you’re forced to wrap then to put most things on their own lines. I’m fairly sure that Prettier stems from [Wadler’s algorithm from “A prettier printer”](https://homepages.inf.ed.ac.uk/wadler/papers/prettier/prettier.pdf
) but I haven’t read the paper enough to confidently state that.

Overally I was going for consistency - for example, in this file, there were `List.of` calls which spanned multiple lines even before formatting, and they have one argument per line. So having one rule for method arguments and one for other types of expressions isn't especially consistent.

Reply:
Thanks for sharing the background here. I understand now why and I think it is ok.
Also I didn't feel strongly about this comment than the first comment.